src/renderer/components/PresetListItem.tsx

```diff
@@ -0,0 +1,88 @@
+import { useRef, useState } from 'react';
+import { useMouseDownOutside } from 'renderer/utils/utils';
+import IconButton, { IconName } from 'renderer/widgets/IconButton';
+import TextInput from 'renderer/widgets/TextInput';
+
+export const formatPresetName = (s: string) => {
```

Comment:
This doesn't seem like the right place to put this function. Perhaps move this to a utils file?